qiskit/__init__.py

```diff
@@ -21,7 +23,8 @@
 from qiskit.circuit import ClassicalRegister
 from qiskit.circuit import QuantumRegister
 from qiskit.circuit import QuantumCircuit
-from .tools.compiler import (compile, execute)
+from qiskit.tools.compiler import (compile, execute)  # TODO remove after 0.8
```

Comment:
The user who wanted to compile a circuit previously accessed the `compile()` function from the global qiskit namespace. 

But now the replacement functions (`transpile`, `assemble`) are under the `qiskit.compile` namespace. So something like `from qiskit import *` will no longer work for this use case. Let's think about whether this is an inconvenience.

Reply:
I like keeping it within the `qiskit.compiler` namespace so I'm gonna leave it like that.